runtime/include/j9cfg.h.ftl

```diff
@@ -70,6 +70,12 @@ extern "C" {
 #define J9VM_OPT_MULTI_LAYER_SHARED_CLASS_CACHE
 #endif
 
+#if defined(J9VM_BUILD_JITSERVER)
+#ifndef JITSERVER_SUPPORT
+#define JITSERVER_SUPPORT
```

Comment:
This feels off to me after looking at it again.  Why are we defining a flag to define a different flag?

The usual way to do this in J9 is to define an `opt_` flag and then use the `J9VM_OPT_JITSERVER` flag everywhere.  I think it would be cleaner to have 1 consistent flag controlled by the `j9.flags` file then to have that flag set this one.

Reply:
Reason to do it this way is `JITSERVER_SUPPORT` is already being used by JIT to enable the related code. And if I remember correctly @keithc-ca wanted to use same macro for VM as well, and this was the easiest way to make it all work (in both cmake and uma build systems).

Reply:
Please open an issue to address this in the future.  There should be a single consistent option used throughout the code base to enable / disable jitserver